package.json

```diff
@@ -71,6 +71,6 @@
     "style-loader": "0.13.1"
   },
   "dependencies": {
-    "@bufferapp/components": "0.2.1"
+    "@bufferapp/components": "0.2.2"
   }
 }
```

Comment:
Using version 0.2.2 to get access to the 'blue' color config for Text component 😄